### DISCUSSION TITLE
ping github.com Nepal not responding

### DISCUSSION THREAD

**Discussion by sujit-baniya** [community/community#41178] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Github doesn't seem to respond from Nepal (Vianet, worldlink or NTC ISP) using doing `ping github.com`

Result: `Request timeout`

**Comment by ghostinhershell** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @sujit-baniya! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, I see that you posted this a couple of weeks ago. Is this issue resolved?
